components/shared/button/Button.test.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import Button, { ButtonType } from './Button';
+
+describe('Button tests', () => {
+  it('No props, empty button', () => {
+    const { queryByTestId } = render(<Button></Button>);
+    const button = queryByTestId('button');
+    expect(button?.className).toContain('bg-white text-gray-700');
+    expect(button?.innerHTML).toBe('');
+  });
+  it('No props with children', () => {
+    const { queryByTestId } = render(<Button>Child element</Button>);
+    const button = queryByTestId('button');
```

Comment:
nie rozumiem co to testuje "on propps with children"
może lepiej: "should render correct button content'

usuń z buttona `data-testid`,
to jest zła praktyka korzystać z tego.
w każdym razie używamy w ostateczności

tu powinienneś skorzystać z `queryByText`

Reply:
zrobione